src/patternfly/components/Check/examples/Check.md

```diff
@@ -102,14 +110,18 @@ If you extend this component or modify the styles of this component, then make s
 | Attribute | Applied to | Outcome |
 | -- | -- | -- |
 | `disabled` | `<input type="checkbox">` | Indicates that the element is unavailable and removes it from keyboard focus. **Required when input is disabled** |
+| `required` | `<input type="checkbox">` | Indicates that the element is required. |
```

Comment:
This is a pre-existing thing, but these attrs can refer to the component class instead of the html element. the only place we need to mention the html element is below when we're documenting the component class.

```suggestion
| `disabled` | `.pf-c-check__input` | Indicates that the element is unavailable and removes it from keyboard focus. **Required when input is disabled** |
| `required` | `.pf-c-check__input` | Indicates that the element is required. |
```